tests/test_compute_utils.py

```diff
@@ -69,7 +69,17 @@ def surface_integrals(grid, q=np.array([1.0]), surface_label="rho"):
             Surface integral of the input over each surface in the grid.
 
         """
-        _, _, spacing, has_endpoint_dupe = _get_grid_surface(grid, surface_label)
+        _, _, spacing, _, _ = _get_grid_surface(grid, grid.get_label(surface_label))
```

Comment:
this uses same logic as master, or what used to be in compute.utils to make sure i didn't make typos when relabling theta to poloidal in compute.utils